### PR TITLE
CLDR-16323 Update code to handle patterns without given, fix charts

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-16323

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by macchiati** [comment]
Including @richgillam so he can see the changes to the person name formatting. Turned out to be pretty straightforward.

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartPersonNames.java [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class ChartPersonNames extends Chart {
     private static final Factory CLDR_FACTORY = CLDRConfig.getInstance().getCldrFactory();
     static final String MAIN_HEADER = "<p>These charts shows the sample person names for different locales, formatted according to the locale's pattern.</p>";
     private static final boolean DEBUG = false;
-    static final String DIR = CLDRPaths.CHART_DIRECTORY + "personNames/";
+    static final String DIR = CLDRPaths.CHART_DIRECTORY + "verify/personNames/";
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartPersonNames.java:18]: what is `verify/` ?

**Review by srl295 — APPROVED** [review]
looks reasonable with a question…

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartPersonNames.java (hunk 1 above)
Comment (macchiati) [reply tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartPersonNames.java:18]: It is one of the chart subfolders.

**Comment by richgillam** [comment]
The changes to the person name formatter look good to me.

**Pull request merged**